site/docs/foundations/typography.mdx

```diff
@@ -5,101 +5,97 @@ sidebar:
   priority: 1
 ---
 
-Typography involves the style and appearance of written language. It plays a crucial role in enhancing readability, establishing hierarchy, and setting a visual tone, resulting in more effective and expressive communication.
+Typography refers to the predefined typographic styles that guide the presentation of written language. Typography is essential for improving readability, creating a clear hierarchy, and establishing a visual tone, all of which contribute to more effective and expressive communication. Each text style in Salt includes a font family, font weight, font size, and line height. The specific values for each is dependent on the applied theme.
 
 ## Font family
 
-The Legacy UITK theme uses the open-source, sans-serif typeface **Open Sans**. The JPM brand theme uses the **Amplitude** font which is in line with updated brand guidelines. Code uses the mono space typeface **PT Mono**.
+The typography foundation offers two font families for UI design: Open Sans and Amplitude, along with PT Mono for code blocks.
 
-For more information about the themes offered in Salt, please refer to the guidance on the [themes page](https://www.saltdesignsystem.com/salt/themes/index).
+<Image
+  src="/img/foundations/typography-fontfamily.png"
+  srcDark="/img/foundations/typography-fontfamily-dark.png"
+  alt=""
+  aria-hidden="true"
+/>
+
+Font families are used differently by each theme. For more information about the themes offered in Salt, please refer to [theme guidance](https://www.saltdesignsystem.com/salt/themes/index).
 
 <Image
-  src="/img/foundations/typography-options.png"
-  alt="Example of Open Sans and Amplitude font"
+  src="/img/foundations/typography-theme.png"
+  srcDark="/img/foundations/typography-theme-dark.png"
+  alt="Example of font weights in JPM Brand theme and Legacy theme"
```

Comment:
@jake-costa This isn't describing font weights, it's focused on font families.

Also my original feedback was that the information is only conveyed in the image. The current alt text doesn't solve that, is that an issue?